cypress/integration/plugins/security-analytics-dashboards-plugin/5_integrations.spec.js

```diff
@@ -75,7 +75,7 @@ describe('Integration tests', () => {
   it('...can navigate to findings page', () => {
     cy.intercept({
       method: 'GET',
-      pathname: '/_plugins/_security_analytics/findings/_search',
+      pathname: '/_dashboards/_plugins/_security_analytics/findings/_search',
```

Comment:
won't this fail the open source tests since _dashboards isn't the base path?

we should be able to configure the cypress.yml file to include the basepath

Reply:
refactored to use regex pattern, so it works everywhere